docs/guide/acquiring_data/fido.rst

```diff
@@ -9,40 +9,107 @@ Federated Internet Data Obtainer...or more usually (and sanely) referred to as F
 `~sunpy.net.Fido` is a unified interface for searching
 and fetching solar physics data irrespective of the underlying
 client or webservice through which the data is obtained, e.g. VSO_,
-JSOC_, etc.  It therefore supplies a single, easy and consistent way to
+JSOC_, etc. It therefore supplies a single, easy and consistent way to
 obtain most forms of solar physics data.
 
 Import
 ------
 
-The `~sunpy.net.Fido` object is in
-`sunpy.net`. It can be imported as follows::
+The `~sunpy.net.Fido` object is in `sunpy.net`.
+It can be imported as follows::
 
     >>> from sunpy.net import Fido, attrs as a
 
+Search Attributes
+-----------------
+
+To search for data with `~sunpy.net.Fido, you need to specify attributes to search against.
+The range of attributes are found in the `attrs <sunpy.net.attrs>` submodule.
+Examples of these attributes are:
+
+- `a.Time <sunpy.net.attrs.Time>`
+- `a.Instrument <sunpy.net.attrs.Instrument>`
+- `a.Wavelength <sunpy.net.attrs.Wavelength>`
+
+whereas some of these attributes are client specific, and are found under `attrs.vso <sunpy.net.vso.attrs>` and `attrs.jsoc <sunpy.net.jsoc.attrs>`.
+
+In to each attribute you have to provide a value to use::
+
+    >>> a.Time('2012/3/4', '2012/3/6'), a.Instrument.lyra
+    (<sunpy.net.attrs.Time(2012-03-04 00:00:00.000, 2012-03-06 00:00:00.000, )>, <sunpy.net.attrs.Instrument(LYRA: Lyman Alpha Radiometer is the solar UV radiometer on board
+    Proba-2.) object at ...>)
+
+
+For attributes that have no fixed selection of values (``Time`` for example) you will have to provide the range you require.
+However, for attributes that have a fixed range of **known** values, it is possible to list all these values.
+**Please note that each list is not exhaustive.**
+
+Using ``Instrument`` as the first example, if you print the object::
+
+    >>> print(a.Instrument)
+    sunpy.net.attrs.Instrument
+    <BLANKLINE>
+    Specifies the Instrument name for the search.
+    <BLANKLINE>
+           Attribute Name          Client          Full Name                                           Description
+    --------------------------- ----------- ------------------------ --------------------------------------------------------------------------------
+    aia                         VSO         AIA                      Atmospheric Imaging Assembly
+    bbi                         VSO         BBI                      None
+    bcs                         VSO         BCS                      Bragg Crystal Spectrometer
+    bic_hifi                    VSO         BIC-HIFI                 None
+    bigbear                     VSO         Big Bear                 Big Bear Solar Observatory, California TON and GONG+ sites
+    ...
+
+You get a full list of known values, a description and what "Clients" support those values (if you want to use a specific data source).
```

Comment:
This seems to disagree with line 45 above that says "**Please note that each list is not exhaustive.**"

Reply:
Oh, I'm guessing there's a distinction between known values and a non-exhaustive list, never mind.